home/templates/home/index.html

```diff
@@ -89,7 +89,7 @@ <h2 class="subtitle">
                     <a href="https://pt.wikipedia.org/wiki/Feira_de_Santana" target="_blank" rel="noopener noreferrer">
                         Feira de Santana.
                     </a>
-                </h2>
+                </h2>                
```

Comment:
```suggestion
                </h2>
```
Não precisa desse espaço em branco extra aqui. :sweat_smile: